documentation/intro-dylan/source/modules-libraries.rst

```diff
@@ -13,7 +13,8 @@ Simple Modules
 Modules import names (or bindings) from other modules and export names
 for use by other modules. The names that may be imported/exported are
 the module-level (also called "global") variables such as those created
-by ``define variable``, ``define class``, ``define generic``, etc.
+by :drm:`define variable <define_variable>`, :drm:`define class
+<define_class>`, :drm:`define generic <define_generic>`, etc.
```

Comment:
As @cgay points out ... this would be much improved by an improvement to `drmindex.py`.